react-time-track/src/components/close-project-modal.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Modal from './modal';
+
+function CloseProjectModal(props) {
+  return (
+    <Modal isOpen={props.isOpen} onClose={props.onClose}>
+      <div>
+        <h1>Are You Sure?</h1>
+        <p>You are about to delete this project.</p>
+
+        <div>
+          <button onClick={props.onCloseProject} >
```

Comment:
What do you think about using the button customize which is already the list of components in ui.js?